electron/anticheat/titles_map.ts

```diff
@@ -0,0 +1,27 @@
+// this hash contains a mapping of titles as displayed in Epic to titles as
+// displayed in areweanticheatyet.com https://github.com/Starz0r/AreWeAntiCheatYet/blob/master/games.json
+//
+// comparison is case insensitive, so no need to add cases like 'VALORANT': 'Valorant'
+// if the epic title is contained, so no need to add cases like 'Fall Guts': 'Fall Guys: Ultimate Knockout'
+//
+// left is the epic title, right is the areweanticheatyet title
+const TITLES_MAP: { [key: string]: string } = {
+  'Battlefield™ 2042': 'Battlefield 2042',
+  'Back 4 Blood: Standard Edition': 'Back 4 Blood',
+  'Rainbow Six Siege': 'Rainbow Six: Siege',
```

Comment:
And perhaps even the colon signs

Reply:
Same for this, I agree that removing some `:` would remove this lines with the current information, but we still need this mapping for other cases.